src/crates.md

```diff
@@ -0,0 +1,87 @@
+# Crates
+
+The Rand library consists of a family of crates. For common usage, the `rand`
+crate alone will often suffice. Other crates serve as building-blocks and/or
+provide additional functionality.
+
+### rand_core
+
+The [`rand_core`] crate defines the core traits implemented by RNGs. This exists
+as a separate crate with two purposes:
+
+-   to provide a minimal API for defining and using RNGs
+-   to provide tools to aid implementation of RNGs
+
+The [`RngCore`], [`SeedableRng`], [`CryptoRng`] traits and [`Error`] type are
+all defined by this crate and re-exported by the [`rand`] crate.
+
+### rand
+
+The [`rand`] crate is optimised for easy usage of common random-number
+functionality. This has several aspects:
+
+-   the [`rngs`] module provides a few convenient generators
+-   the [`distributions`] module concerns sampling of random values
+-   the [`seq`] module concerns sampling from and shuffling sequences
+-   the [`Rng`] trait provides a few convenience methods for generating
+    random values
+-   the [`random`] function provides convenient generation in a single call
+
+## Distributions
+
+The [`rand`] crate only implements sampling from the most common random
+number distributions: uniform and weighted sampling. For everything else,
+
+-   [`rand_distr`] provides fast sampling from a variety of other distributions,
+    including Normal (Gauss), Binomial, Poisson, UnitCircle, and many more
+-   [`statrs`] is a port of the C# Math.NET library, implementing many of the
+    same distributions (plus/minus a few), along with PDF and CDF functions,
+    the *error*, *beta*, *gamma* and *logistic* special functions, plus a few
+    utilities. (For clarity, [`statrs`] is not part of the Rand library.)
+
+## Generators
+
+### Deterministic generators
+
+The following crates implement [PRNGs]:
+
+-   [`rand_chacha`] provides generators using the ChaCha cipher
+-   [`rand_hc`] implements a generator using the HC-128 cipher
+-   [`rand_isaac`] implements the ISAAC generators
+-   [`rand_pcg`] implements a small selection of PCG generators
+-   [`rand_xoshiro`] implements the SplitMix and Xoshiro generators
+-   [`rand_xorshift`] implements the basic Xorshift generator
```

Comment:
Maybe subdivide this into into generators that are cryptographically secure and generators that aren't.

Is it worth it to mention the `rand_isaac` and `rand_xorshift` generators? I consider them deprecated.

Reply:
The point is just to make a quick overview of crates, not go into the RNGs.

Reply:
Fair enough, in this case the list should indeed be exhaustive.